### PR TITLE
New plant: 3D Lime

### PR DESCRIPTION
**Pull request by warface1234455**
### _**ITS 3D**_

# Document the changes in your pull request
new plant 3d lime
lime can now be mutated to 3d lime
new seed 3d lime

# Why is this good for the game?
3d lime is cooler than 3d orange and it doesnt have reference to any TG HEAD CODER!!!!

# Testing
tested on local

# Spriting
image

# Wiki Documentation
goes into guide to plants

# Changelog

:cl:  warface1234455(code and seed sprite), baiomu (3d lime sprite)
rscadd: new plant 3d lime
tweak: lime can now be mutated to 3d lime
tweak: new 3d lime seed
/:cl:

### CODE REVIEW

**Review by Moltijoe — CHANGES_REQUESTED** [review]
have it split the chems in the 3d lime into the created limes

**Comment by ShoziOffical** [comment]
> have it split the chems in the 3d lime into the created limes

gotta make my life harder

**Comment by Moltijoe** [comment]
> gotta make my life harder

yes

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -33,6 +33,59 @@
...
+		lime.seed.endurance = seed.endurance
+		lime.seed.maturation = seed.maturation
+		lime.seed.production = seed.production
+		lime.seed.yield = seed.yield
+		lime.seed.potency = seed.potency
+		lime.seed.weed_rate = seed.weed_rate
+		lime.seed.weed_chance = seed.weed_chance
+		lime.seed.genes |= seed.genes
+		for(var/datum/reagent/R  in reagents.reagent_list)
+			R.volume /= 2
```
Comment (Moltijoe) [inline code/modules/hydroponics/grown/citrus.dm:82]: why is it only cutting the reagents in half, that would make it duplicate reagents

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/hydroponics/grown/citrus.dm:82]: i tried dividing to the total splitted limes but the reagents would be real close to 0 which would result in none

Comment (ShoziOffical) [reply code/modules/hydroponics/grown/citrus.dm:82]: like 0.00000000000032e-7

Comment (ShoziOffical) [reply code/modules/hydroponics/grown/citrus.dm:82]: either way doesnt matter, because botanist can already duplicate chems with the 3d lime

**Comment by MajManatee** [comment]
Dumb to have gotten into a hissy over the 3D orange to begin with

**Review by cuackles — APPROVED** [review]
mmmmmmmm lemon

**Comment by ShoziOffical** [comment]
> mmmmmmmm lemon

lime

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**